go.mod

```diff
@@ -19,9 +19,11 @@ require (
 	github.com/vincent-petithory/dataurl v0.0.0-20191104211930-d1553a71de50
 	go.etcd.io/etcd v0.0.0-20200401174654-e694b7bb0875
 	google.golang.org/grpc v1.26.0
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
Suggest we switch to JSON because:

1. We don't need to add a new dependency
2. Tools consuming the data file are more likely to support JSON than YAML

Reply:
If we are going to ask the users to inspect the backupenv files to know which one is latest, the yaml files are more readable for the end-users than json.

Reply:
But I changed to json anyways.

Reply:
Looks like you might need to run `go mod vendor` again

Reply:
and/or `go mod tidy`